projects/go-lib/src/lib/components/go-datepicker/date-adapter.ts

```diff
@@ -10,13 +11,14 @@ export class DateAdapter {
 
   public setLocale(locale: string): void {
     this._locale = locale;
+    this._timezone = Intl.DateTimeFormat().resolvedOptions().timeZone;
     this.setMonthNames();
     this.setDateNames();
     this.setDayOfWeekNames();
   }
 
   public getYearName(year: number): string {
-    const format: Intl.DateTimeFormat = new Intl.DateTimeFormat(this._locale, {year: 'numeric', timeZone: 'utc'});
+    const format: Intl.DateTimeFormat = new Intl.DateTimeFormat(this._locale, {year: 'numeric', timeZone: this._timezone});
```

Comment:
I think by default if we don't pass in timeZone here it would be the same as using Intl.DateTimeFormat().resolvedOptions().timeZone. So instead of switching it here to save the timezone as a variable can we just remove the timeZone from all of these?

Reply:
@StevenUlmer yep, seems to be still working fine. Might of misread the docs and thought it defaulted to UTC.